### PR TITLE
added test for pagination to this endpoint

### PR DESCRIPTION
**Pull request by parrfolio**

### CODE REVIEW

File: src/users/collection/getItemsInFolderForUser.test.js [hunk 1]
```diff
@@ -22,8 +22,13 @@ describe('Users - Collection - getItemsInFolderForUser', () => {
     })
   })
   it('should get items in a folder in a user\'s collection from its username and a `folderId`', async () => {
-    const data = await client.getItemsInFolderForUser(username, folderId)
+
+    const data = await client.getItemsInFolderForUser({ username, folderId, pagination: { perPage: 1 } })
```
Comment (aknorw) [inline src/users/collection/getItemsInFolderForUser.test.js]: This won't pass as you are passing an object to the function (see other PR).

File: src/users/collection/getItemsInFolderForUser.test.js [hunk 2]
```diff
@@ -45,4 +50,18 @@ describe('Users - Collection - getItemsInFolderForUser', () => {
     sort.should.be.equal(by)
     sort_order.should.be.equal(order)
   })
+  it('should accept pagination', async () => {
+    const page = 1
+    const perPage = 3
+    const data = await client.getInventory({
```
Comment (aknorw) [inline src/users/collection/getItemsInFolderForUser.test.js]: You didn't change the function name when copying-pasting, so this test does not affect `getItemsInFolderForUser`

File: src/users/collection/getItemsInFolderForUser.test.js (hunk 1 above)
Comment (parrfolio) [reply src/users/collection/getItemsInFolderForUser.test.js]: removed object, oops

**Comment by parrfolio** [comment]
Moved all this to #4

**Comment by aknorw** [comment]
I'm closing this, continuing on #4

**Pull request closed without merging**